src/test/scala/cromwell/OptionalParamWorkflowSpec.scala

```diff
@@ -35,4 +37,35 @@ class OptionalParamWorkflowSpec extends CromwellTestkitSpec("OptionalParamWorkfl
       )
     }
   }
+
+  "A workflow with an optional parameter that has a prefix inside the tag" should {
+    "not include that prefix if no value is specified" in {
+      val wf = """
+         |task find {
+         |  command {
+         |    find ${File root} ${"-name " pattern?}
+         |  }
+         |}
+         |
+         |workflow wf {
+         |  call find
+         |}
+       """.stripMargin
+      val ns = WdlNamespace.load(wf, BackendType.LOCAL)
```

Comment:
Well I'm not opposed to moving it there, though usually I put stuff in `SampleWdl` when I intend to run it through the actor system, but not necessarily for just parsing.  SampleWdl makes you define things like `RawInputs` which felt awkward for me considering I wasn't going to use them.  I know I've done the inlining in other places in the past and it works well especially for very targeted syntax checks.